src/connection.ts

```diff
@@ -996,7 +996,7 @@ export class Connection extends EventEmitter {
 
     // Determine exchange rate from amount that arrived
     this.exchangeRate = responsePacket.prepareAmount.dividedBy(sourceAmount)
-    this.debug(`determined exchange rate to be: ${this.exchangeRate}`)
+    this.log.trace(`determined exchange rate to be: ${this.exchangeRate}`)
```

Comment:
debug level would be good for this since the exchange rate determination is key to the connection.